proto-compiler/README.md

```diff
@@ -4,7 +4,18 @@ The `ibc-proto-compiler` is a simple command-line tool to automate the compilati
 
 ## Usage
 
-### Clone the Cosmos SDK
+### Nix Shell
+
+There is a script wrapper provided by nix that takes care of the `protoc` dependency imposed by `prost-build` and provides a shell wrapper that handles cloning the cosmos go dependencies.
+
+Instruction on how to install nix can be found [here](https://nixos.org/download.html)
+
+```shell
+$ nix-shell # you can run "nix develop" if you have experimental support for flakes
+$ compile
+```
+
+### Using Cargo
```

Comment:
I'll admit, switching `Clone the Cosmos SDK` to `Using Cargo` is taking a bit of a liberty. But I personally found it a bit confusing to have `Clone the Cosmos SDK` as a header, when that is really just 1 step of the process, and it is handled by a rust program anyways, `Using Cargo` felt more semantic. Happy to change this back if you disagree!